youtube_dl/options.py

```diff
@@ -757,6 +757,14 @@ def _scrub_eq(o):
         metavar='FORMAT', dest='recodevideo', default=None,
         help='Encode the video to another format if necessary (currently supported: mp4|flv|ogg|webm|mkv|avi)')
     postproc.add_option(
+        '--slice-start',
+        metavar='START_TIME', dest='slicestart', default=None,
```

Comment:
`metavar` should be `TIME`, it should also support various formats (use `parse_duration`).